### PR TITLE
DP-790 - Localizing Organisation registration process & error summary

### PR DESCRIPTION
**Pull request by dpatel017**
Add translations for organisation registration

### CODE REVIEW

File: Frontend/CO.CDP.OrganisationApp/Pages/Registration/JoinOrganisation.cshtml [hunk 1]
```diff
@@ -36,23 +37,23 @@
                 <dl class="govuk-summary-list">
```
Comment (andymantell) [inline Frontend/CO.CDP.OrganisationApp/Pages/Registration/JoinOrganisation.cshtml:37]: Missed "User details" heading just above this line

File: Frontend/CO.CDP.OrganisationApp/Pages/Registration/OrganisationAlreadyRegistered.cshtml [hunk 2]
```diff
@@ -13,7 +14,9 @@
     }
 }
 
-<a href="@backLink" class="govuk-back-link">Back</a>
+<a href="@backLink" class="govuk-back-link">
+    @StaticTextResource.Global_Back
+</a>
 
 <main class="govuk-main-wrapper" id="main-content">
```
Comment (andymantell) [inline Frontend/CO.CDP.OrganisationApp/Pages/Registration/OrganisationAlreadyRegistered.cshtml:21]: Text below this line is not translated

File: Frontend/CO.CDP.OrganisationApp/Pages/Registration/OrganisationEmail.cshtml [hunk 3]
```diff
@@ -39,7 +40,7 @@
                             </label>
                         </h1>
 
-                        <div id="hint" class="govuk-hint">Where possible, use a shared inbox that other team members can access</div>
+                        <div id="hint" class="govuk-hint">@StaticTextResource.OrganisationRegistration_EnterOrganisationEmail_Hint</div>
 
                         <div class="govuk-form-group govuk-!-width-two-thirds @(emailAddressHasError ? "govuk-form-group--error" : "")">
                             @if (emailAddressHasError)
```
Comment (andymantell) [inline Frontend/CO.CDP.OrganisationApp/Pages/Registration/OrganisationEmail.cshtml:46]: There's a few instances of `<span class="govuk-visually-hidden">Error:</span>` that aren't translated

File: Services/CO.CDP.Localization/StaticTextResource.cy.resx [hunk 4]
```diff
@@ -144,4 +147,19 @@
...
+  </data>
+  <data name="Global_ErrorSummary_Heading" xml:space="preserve">
+    <value>Mae yna broblem</value>
+  </data>
+  <data name="Global_EmailAddress_Error" xml:space="preserve">
+    <value>Rhowch gyfeiriad e-bost yn y fformat cywir, fel enw@enghraifft.com</value>
+  </data>
+  <data name="Organisation_EnterOrganisationName_ErrorMessage" xml:space="preserve">
+    <value>Rhowch enw’r sefydliad</value>
+  </data>
```
Comment (andymantell) [inline Services/CO.CDP.Localization/StaticTextResource.cy.resx:164]: Are these from Google translate? What's `enw@enghraifft.com` ?

**Review by andymantell — CHANGES_REQUESTED** [review]

File: Frontend/CO.CDP.OrganisationApp/Pages/Registration/OrganisationAlreadyRegistered.cshtml (hunk 2 above)
Comment (dpatel017) [reply Frontend/CO.CDP.OrganisationApp/Pages/Registration/OrganisationAlreadyRegistered.cshtml:21]: Done

File: Frontend/CO.CDP.OrganisationApp/Pages/Registration/OrganisationEmail.cshtml (hunk 3 above)
Comment (dpatel017) [reply Frontend/CO.CDP.OrganisationApp/Pages/Registration/OrganisationEmail.cshtml:46]: Done

File: Services/CO.CDP.Localization/StaticTextResource.cy.resx (hunk 4 above)
Comment (dpatel017) [reply Services/CO.CDP.Localization/StaticTextResource.cy.resx]: Done

File: Frontend/CO.CDP.OrganisationApp/Pages/Registration/JoinOrganisation.cshtml (hunk 1 above)
Comment (dpatel017) [reply Frontend/CO.CDP.OrganisationApp/Pages/Registration/JoinOrganisation.cshtml:37]: Done

**Review by andymantell — DISMISSED** [review]

**Review by andymantell — APPROVED** [review]

**Review by JBaigGoaco — APPROVED** [review]

File: Frontend/CO.CDP.OrganisationApp/Pages/Registration/BuyerSelectDevolvedRegulation.cshtml.cs [hunk 5]
```diff
@@ -10,7 +11,7 @@ public class BuyerSelectDevolvedRegulationModel(ISession session) : Registration
     public override string CurrentPage => BuyerSelectDevolvedRegulationPage;
 
     [BindProperty]
-    [NotEmpty(ErrorMessage = "Select a devolved region")]
+    [NotEmpty(ErrorMessage = nameof(StaticTextResource.OrganisationRegistration_BuyerSelectDevolvedRegulation_ValidationErrorMessage))]
```
Comment (andymantell) [inline Frontend/CO.CDP.OrganisationApp/Pages/Registration/BuyerSelectDevolvedRegulation.cshtml.cs:14]: I think this won't work - for me the `NotEmpty` validator is not accepting this style of doing things which would mean this validation error message literally shows on screen as `OrganisationRegistration_BuyerSelectDevolvedRegulation_ValidationErrorMessage`

Comment (andymantell) [reply Frontend/CO.CDP.OrganisationApp/Pages/Registration/BuyerSelectDevolvedRegulation.cshtml.cs:14]: @dpatel017 I have sorted this out on my branch in 3087763c94c35c2367c521a947a065e0d0119753

If you `git cherry-pick 3087763c94c35c2367c521a947a065e0d0119753`  to bring it across to yours, and then you will need to do the following as well:

```suggestion
    [NotEmpty(ErrorMessageResourceName = nameof(StaticTextResource.OrganisationRegistration_BuyerSelectDevolvedRegulation_ValidationErrorMessage), ErrorMessageResourceType = typeof(StaticTextResource))]
```

**Review by SDGoaco — APPROVED** [review]
The Ticket is working as Expected in the  Local environment and Dev Environment

**Review by jakzal — APPROVED** [review]

**Pull request merged**